wiki/Tournaments/Official_support/en.md

```diff
@@ -161,23 +176,32 @@ Badge images that do not meet these standards will be refused with given reasoni
 
 In exceptionally rare cases, a tournament may be granted the privilege of awarding badges to its top three finalists. A tournament that has received this distinction is recognised for attaining the highest level of production quality and organisation possible — this is given exclusively at the discretion of the osu! team. If you believe your tournament meets the expectations to receive this distinction, you may request it when submitting your initial badge request.
 
+Note: podium finishing/finalist badge prizes are only awarded if the tournament in question does not have any restrictions in place for participation, such as being limited to a specific rank range or a specific geographic region.
+
 Examples of exceptional tournaments that have met these expectations:
 
 - [Corsace Open](https://osu.ppy.sh/community/forums/topics/1607468)
 - [Expert Global Taiko Showdown](/wiki/Tournaments/GTS/EGTS_2022)
 - [Springtime osu!mania Free-for-all Tournament](/wiki/Tournaments/SOFT/6)
 
-### Requesting main-menu banner support
+### Requesting extra visibility support (main-menu banners and news posts)
 
-Long-running community tournaments may qualify for in-game main-menu banner support. It's at the osu! team's discretion to decide if your tournament can be advertised in-game, as there are many factors to consider. The minimum requirements to request in-game advertising are as follows:
+Long-running community tournaments may qualify for additional support to enhance their visibility to the playerbase, such as receiving main-menu banners in-game and publication via news posts on the osu! website. The minimum requirements to request in-game advertising are as follows:
```

Comment:
this kind of wording implies that there are more ways of requesting advertising support when that's not really intended, which feels slightly weird to me

unless you mean using https://osu.ppy.sh/community/tournaments for community tournaments, which hasn't been part of discussions to my knowledge

```suggestion
Long-running community tournaments may qualify for additional support to enhance their visibility to the playerbase, namely receiving main-menu banners in-game and publication via news posts on the osu! website.

The minimum requirements to request in-game or website advertising support are as follows:
```